task/01-strings-tasks.js

```diff
@@ -201,9 +196,23 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
-}
+  let rectangle = "";
```

Comment:
Please, try to solve it without using let

Reply:
Sorry? Don't get it

Reply:
You used here `let` for the rectangle variable, try to rewrite this function with `const`